src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py

```diff
@@ -187,6 +187,14 @@ def register_options(cls, register):
     register('--suggest-missing-deps', type=bool,
              help='Suggest missing dependencies on a best-effort basis from target\'s transitive'
                   'deps for compilation failures that are due to class not found.')
+    
+    register('--missing-deps-not-found-msg', type=str,
+             help='The string that should be printed when missing-deps-suggest can\'t find any '
+                  'targets for the classes not found during compilation. This should likely '
+                  'include a link to documentation about dependency management.',
+             default='Missing-deps-suggest could not find the dependencies needed to compile '
```

Comment:
I don't think folks will recognize the name of the tool here, so maybe just `"Pants could not find..."`